### PR TITLE
Profile routing

### PR DESCRIPTION
**Pull request by keonp**
oh my

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,6 +26,8 @@ function App() {
             }
           ></Route>
 
+          <Route path="/profile/:uid" element={<UserProfile/>}/>
```
Comment (chxwcodes) [inline src/App.js:30]: ooo dynamic urls 👀

File: src/components/UserProfile.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    }) : <ErrorPage />
+                }
+            </ul>
+
+            <div className='profileButtons'>
+                <Link to="/">
+                    <button>Back</button>
+                </Link>
+                <Link to="/">
+                    <button onClick={(e) => handleUserAccountDeletion(e)}
```
Comment (chxwcodes) [inline src/components/UserProfile.js:107]: nuke btn 👀

**Review by chxwcodes — APPROVED** [review]
wowowowoow

**Comment by pinggyu** [comment]
✨AMAZE✨

**Pull request merged**